public/app/features/dashboard/components/GenAI/GenAIDashDescriptionButton.tsx

```diff
@@ -19,7 +19,6 @@ const DESCRIPTION_GENERATION_STANDARD_PROMPT =
   'If the dashboard has no panels, the description should be "Empty dashboard"\n' +
   'There should be no numbers in the description except where they are important.\n' +
   'The dashboard description should not have the dashboard title or any quotation marks in it.\n' +
-  'The description should be, at most, 140 characters.\n' +
```

Comment:
I wonder if we should keep the description limit at least for now, given the feedback about the duration - if we got that feedback having a limit, I would imagine it will only get worse without 🤔

Reply:
what limit do you think is reasonable? Currently for panel description we limit to 200 characters so maybe 250-300?

Reply:
done - made it 300 character limit 👍